aten/src/ATen/native/cpu/UnaryOpsKernel.cpp

```diff
@@ -0,0 +1,132 @@
+#include "ATen/native/cpu/UnaryOpsKernel.h"
+#include "ATen/Dispatch.h"
+#include "ATen/Parallel.h"
+#include <iostream>
+#include <cmath>
+#include "ATen/native/cpu/Vec256.h"
+
+namespace at {
+namespace native {
+
+using namespace vec256;
+
+// This modifies arr in place with given OP
+template <class scalar_t, template <class> class VOP, template <class> class SOP, CPUCapability C>
+inline void kernel_(scalar_t *arr_out, const scalar_t *arr_in, size_t start, size_t end) {
+  // Use all 16 registers.
+  Vec256<scalar_t> a[8];
+  size_t width =
+      256 / sizeof(scalar_t); // primitives per 256 bytes (two cache lines)
```

Comment:
256 bytes is not two cache lines. It's four cache lines.  (Cache lines are 64 bytes).